src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -16,6 +16,8 @@ import UploadDetachedFileValidation from './UploadDetachedFileValidation.jsx';
 
```

Comment:
State changes fire off the page to re-render. using only param changes may result in the page refusing to load.